command_server/command_server.py

```diff
@@ -37,7 +38,9 @@ def __init__(self, host=None, port=None, loop=None):
         logging.info('Init Server on host %s:%s' % (host, port))
         self._loop = loop or asyncio.get_event_loop()
         #TODO: MOve debug mode to config file.
-        self._server = self._loop.create_server(lambda: HttpRequestHandler(),
+        self._app = web.Application(loop=loop)
```

Comment:
It is not my wish) we just can't process all kind of requests in one way, that's all